libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -935,22 +950,18 @@ private static DateTime ParseTimestamp(string timeStamp)
             {
                 // Math
                 { ExpressionType.Element, new ExpressionEvaluator(ExtractElement, ReturnType.Object,ValidateBinary) },
-                { ExpressionType.Add, Numeric(args => args[0] + args[1]) },
-                { ExpressionType.Subtract, Numeric(args => args[0] - args[1]) },
-                { ExpressionType.Multiply, Numeric(args => args[0] * args[1]) },
-                { ExpressionType.Divide,
-                    new ExpressionEvaluator(ApplySequence(args => args[0] / args[1],
-                    (value, expression) => {
-                        var error = VerifyNumber(value, expression);
-                        if (error == null && Convert.ToDouble(value) == 0.0)
-                        {
-                            error = $"Cannot divide by 0 from {expression}";
-                        }
-                        return error;
-                    }), ReturnType.Number, ValidateNumber) },
+                { ExpressionType.Add, MultivariateNumeric(args => args[0] + args[1]) },
+                { ExpressionType.Subtract, new ExpressionEvaluator(Apply(args => args[0] - args[1], VerifyNumber),
+                        ReturnType.Number, ValidateBinaryNumber)},
```

Comment:
Subtract and Divide can also accept multiple parameters given a-b-c -> subtract(a, b, c) and a/b/c -> divide(a, b, c) are reasonable.

Reply:
fixed it in the lastest commit